test/e2e/capv_test_utils.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package e2e
 
 import (
-	"crypto/sha1" //nolint:gosec
+	"crypto/sha1" // nolint:gosec
```

Comment:
any reason not to just bump this to sha256?

Reply:
this was just a bulk find and replace, didn't even spot this. changing it now.